src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -337,11 +329,14 @@ void TxDetailDialog::onOutputsClicked()
             // If the there is a model tx, then this is a confirmation dialog
             if (tx) {
                 const QList<SendCoinsRecipient>& recipients = tx->getRecipients();
+                unsigned int sffa = tx->subtractFeeFromRecipents();
+                CAmount rcp_fee = (sffa > 0) ? (tx->getTransactionFee() / sffa) : 0;
                 for (int i = 0; i < recipients.size(); ++i) {
                     const auto& recipient = recipients[i];
+                    CAmount rcp_amt = recipient.amount - (recipient.fSubtractFee ? rcp_fee : 0);
```

Comment:
nit: better to use `camelCase` instead of `snake_case`. `rcp_fee ->  rcpFee`, `rcp_amt -> rcpAmt`.
At least for now, we are using camel case almost everywhere in the sources with the only exception of the python test suite where we use snake case (we should talk about it and update the code styling docs in the future..).